miio/gateway/devices/subdevices.yaml

```diff
@@ -618,6 +618,22 @@
       unit: Watt
       get: get_property_exp
 
+- zigbee_id: lumi.plug.mmeu01
+  model: ZNCZ04LM
+  type_id: -2
```

Comment:
Is it normal for type id to be negative? Otherwise LGTM :+1:

Reply:
No but I don't know the type id and could not find it on the internet, so I set it to a negative value.
Someone who owns the device and a chinese version of the Hub schould get a warning about the device not beeing known and a message that he schould report it here.
Than we can change the -2 to the correct value.

Reply:
Ok :+1: